lib/compilers/dotnet.ts

```diff
@@ -64,16 +66,14 @@ class DotNetCompiler extends BaseCompiler {
         this.langVersion = this.compilerProps<string>(`compiler.${this.compiler.id}.langVersion`);
 
         this.corerunPath = path.join(this.clrBuildDir, 'corerun');
-        this.crossgen2Path = path.join(this.clrBuildDir, 'crossgen2', 'crossgen2.dll');
+        this.crossgen2Path = path.join(this.clrBuildDir, 'crossgen2', 'crossgen2');
```

Comment:
is it OK to unconditionally drop the `.dll` extension here? edit: I see you check for the presence of either below.

Reply:
Yeah. This is due to an upstream breaking change where it now builds `crossgen2` with AOT directly instead of `crossgen2.dll`. So, we have to check and fallback to `dotnet crossgen2.dll` if `crossgen2` does not exist .

Reply:
Got it.